pkg/kubectl/cmd/util/helpers.go

```diff
@@ -455,19 +454,6 @@ type ValidateOptions struct {
 	SchemaCacheDir   string
 }
 
-func ReadConfigDataFromReader(reader io.Reader, source string) ([]byte, error) {
```

Comment:
Although I agree this is not used anywhere, but since this is a exported function it might be used outside of k8s repo and removing it might break users. I'd suggest adding a comment here about deprecating this, but we can't remove it, yet.

Reply:
yes, it makes sense. I'll put a comment instead. thanks.